org.rdkit.knime.types/rdkit-chemsrc/org/rdkit/knime/types/RDKitMolValueRenderer.java

```diff
@@ -280,9 +273,9 @@ else if (value instanceof SmartsValue) {
 							RDKFuncs.sanitizeMol(tmol);
 							omol.delete();
 							omol = tmol;
-						} 
+						}
 						catch (final Exception ex) {
-							trySanitizing = false;					
+							trySanitizing = false;
```

Comment:
In the future it would be nice to avoid the gratuitous formatting changes that are sprinkled throughout this file.
If we think the files should be reformatted it should be done in one step.
Otherwise please avoid it

Reply:
Yes, removing the trailing spaces was not the greatest idea I have ever had.

Reply:
I discussed another change and reverted something back that Paolo had removed. Now I am fine with it after doing also visually the testing.